internal/testplanet/planet.go

```diff
@@ -102,6 +102,7 @@ func NewWithLogger(log *zap.Logger, satelliteCount, storageNodeCount, uplinkCoun
 
 	// init Satellites
 	for _, node := range planet.Satellites {
+		ctx := context.Background()
```

Comment:
Nothing in New should require a context. If it does then the logic is in the wrong place.

See "Service Lifecycle Simplification".

Reply:
These tests need a consistent context to load the Cache and Kademlia from, or else the cache is an empty pointer and I get a goroutine error. I can hoist the Context creation up in to a higher function, but this feels like a problem with the use of Context.

Reply:
That feels like it should be a different PR.

Reply:
All Egon is saying is 'move stuff with ctx to startup()'
I agree, there's already a loop in Startup() for Kademlia's StartRefresh() there.  At minimum, its wonky to have this into two different places.

Reply:
Okay, I moved all the context stuff to be passed down through args from the actual test setup.